contrib/ros/robots/sawyer.py

```diff
@@ -0,0 +1,123 @@
+"""
+Sawyer Interface
+"""
+
+import intera_interface
+import numpy as np
+import rospy
+from intera_core_msgs.msg import JointLimits
```

Comment:
Please order alphabetically

Reply:
Not sure how to sort from ... statement and import ... statement

Reply:
See comment above. Ignore the `from` and sort only based on package names (and then by class/function names, if applicable)